packages/form-js-viewer/src/render/components/util/dateTimeUtil.js

```diff
@@ -146,15 +146,10 @@ export function parseIsoTime(isoTimeString) {
 }
 
 export function serializeDate(date) {
-  var d = new Date(date),
-    month = '' + (d.getMonth() + 1),
-    day = '' + d.getDate(),
-    year = d.getFullYear();
-
-  if (month.length < 2) month = '0' + month;
-  if (day.length < 2) day = '0' + day;
-
-  return [year, month, day].join('-');
+  const year = date.getFullYear();
+  const month = String(date.getMonth() + 1).padStart(2, '0');
+  const day = String(date.getDate()).padStart(2, '0');
+  return `${year}-${month}-${day}`;
```

Comment:
You can use [`toISOString()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString) and split by `T` and get the first element

Reply:
The behavior is different, toISOString converts to UTC which would not extract the right date if the timezone difference is large enough.